server/Cargo.toml

```diff
@@ -17,3 +17,6 @@ serde = "1.0.164"
 futures = "0.3.28"
 secp256k1 = "0.27.0"
 sha3 = "0.10.8"
+tokio-postgres = "0.7.8"
+lapin = "2.2.1"
```

Comment:
Omg apparently a lapin is a castrated rabbit lol

Reply:
let's go with the second meaning for PG rating! https://www.merriam-webster.com/dictionary/lapin